packages/core/src/components/RadioButton/RadioButton.tsx

```diff
@@ -11,9 +12,9 @@ export type RadioButtonProps = {
   selected?: boolean;
   disabled?: boolean;
   color?: string;
-  value?: string;
+  value: string | number;
```

Comment:
any reason you've removed the optional part of this? i don't think it will always include the value prop